### PR TITLE
Use proper bmap file for tests

### PR DESCRIPTION
**Pull request by dlech**
The previous bmap file mapped the entire image in a single block. This didn't really test if bmaps are working since there were no holes in the map.

Created a new bmap of the etcher-test.img using `bmap create` and added it. Updated one test for the new number of block maps and checksums. Other tests are failing, likely because of an existing bug.

### CODE REVIEW

**Comment by dlech** [comment]
See https://github.com/balena-io/etcher/issues/2754.

There is some bug with bmaps, but I haven't figured it out yet.

**Comment by dlech** [comment]
Found the problem. There was an off-by-one error when calculating the size of a block from a bmap file.

File: tests/metadata-zip.spec.ts [hunk 1]
```diff
@@ -78,14 +78,18 @@ describe('metadata zip', function() {
 	const blockMap = parse(
 		outdent`
 			<?xml version="1.0" ?>
-			<bmap version="1.3">
+			<bmap version="2.0">
```
Comment (lurch) [inline tests/metadata-zip.spec.ts]: Is it worth creating separate test files, to check that **both** bmap versions are correctly supported? Or does that lie outside the scope of etcher-sdk? :man_shrugging:

Comment (dlech) [reply tests/metadata-zip.spec.ts]: I thought about it, but 2.0 has been around since 2013.

**Comment by zvin** [comment]
Moved here #113 so the CI can run.
Will merge asap.
Thanks @dlech !

**Pull request closed without merging**